src/datasets/arrow_dataset.py

```diff
@@ -2865,6 +2865,7 @@ def to_json(
         self,
         path_or_buf: Union[PathLike, BinaryIO],
         batch_size: Optional[int] = None,
+        num_proc: Optional[int] = None,
```

Comment:
could you update the docstring please to document `num_proc` ?

Reply:
I'll update the docstring. Should we also update the tests?